### DISCUSSION TITLE
[Feature] Mobile app supports multiple domains

### DISCUSSION THREAD

**Discussion by chenzn1** [immich-app/immich#7817] [Feature Request]
### The feature

I created a cloudflare tunnel for immich, but it's very slow sometimes. 
So if mobile app support multiple domains and sort by priority, i can setup the LAN address and cloudflare tunnel address. When I'm at home, the app uses the LAN address, and when I'm outside, it uses the cloudflare tunnel address.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of https://github.com/immich-app/immich/issues/308
